src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -50,17 +43,10 @@ const SearchResults = () => {
       query = searchParams.get("query"),
       sort = searchParams.get("sort");
 
-	const getPageSize = (filter) => {
-		switch (filter) {
-			case "account":
-			case "course": {
-				return 12;
-			}
-			default: {
-				return 4;
-			}
-		}
-	};
+  const getPageSize = (filter) => filter ? {
+    "account": 12,
+    "course": 12,
+  } : 4;
```

Comment:
lol this wont actually work how you want it to. you need to index the object like `{...}[filter]`